internal/mptrie/store/open.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 var (
+	nodeGetTotal = 0
+	nodeGetDB = 0
```

Comment:
Please remove. For time, we can calculate and print the time taken. For low level metrics like these, we need to use prometheus.

Reply:
ok